server/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node src/server.js",
+    "dev": "nodemon src/server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@koa/cors": "^3.0.0",
+    "@koa/router": "^8.0.2",
+    "@types/axios": "^0.14.0",
```

Comment:
Hi there, Rankun! Are you using this for something? I don't see any typescript in the project

Reply:
Hi John, thank you for taking time reviewing my code. I was trying to make my vscode to show me the function signatures. Usually I’d put it inside devDependencies, but you are right it’s not necessary.

Reply:
Resolved: fixed, https://github.com/rankun203/test-fullstack/commit/ab0c475c3be5cad7529fc2e1abb15a0811e19bea